src/lib.rs

```diff
@@ -300,7 +300,11 @@ impl PluginGroup for DefaultXrPlugins {
                     ..default()
                 }),
                 #[cfg(target_os = "android")]
-                primary_window: None, // ?
+                primary_window: Some(Window {
+                    resizable: false,
+                    mode: WindowMode::BorderlessFullscreen,
+                    ..default()
+                }),
```

Comment:
I'm not sure this is actually necessary, I don't have hardware on me to test without it right now

Reply:
why did u change this? 

The reason we removed the window on android originally was because having it slowed down vr significantly, is having the window exist on android now a requirement?

Reply:
I don't think so, it's only in here because I was trying a bunch of stuff to fix a bug and I never tested it without this